client/src/app-components/ticket-viewer.js

```diff
@@ -118,7 +118,7 @@ class TicketViewer extends React.Component {
                 <div className="ticket-viewer__comments">
                     {ticket.events && ticket.events.map(this.renderTicketEvent.bind(this))}
                 </div>
```

Comment:
Before:
![image](https://user-images.githubusercontent.com/59095036/98287772-9ae86e00-1f84-11eb-8ea2-dbddadead528.png)

After:
![image](https://user-images.githubusercontent.com/59095036/98287805-a6d43000-1f84-11eb-92a1-7cbbdfbf8a1c.png)

Reply:
Hi,
I found it interesting, but could have an option to respond and close the ticket. Similar to what we have here on GitHub 8-)